doc/cascadia/profiles.schema.json

```diff
@@ -2746,6 +2747,11 @@
           "description": "When set to true, when opening a new tab or pane it will get reloaded environment variables.",
           "type": "boolean"
         },
+        "compatibility.input.forceVT": {
```

Comment:
can you update "experimental.input.forceVT" with:
- "description": "Deprecated. Please use \"compatibility.input.forceVT\" instead."
- "deprecated": true

That way, we'll direct people to use this new one.

Reply:
Good catch. I've pushed a commit that incorporates the above, and done a broader review of all the deprecated settings and updated them with consistent language for the description. There's only 9 deprecated settings, so not much effort.